docs/mevblocker/concepts/endpoints.md

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 2
+---
+
+# MEV Blocker Endpoints
+
+As a MEV Blocker user, you can choose across multiple endpoints to send transactions to depending on your needs.
+
+![Endpoints](/img/mevblocker/Endpoints_protection.png)
+
+- **[Fast](https://rpc.mevblocker.io/fast)** - focused on getting your transaction included onchain as fast as possible while forwarding you a rebate.
+- **[No reverts](https://rpc.mevblocker.io/noreverts)** - focused on getting your transaction included onchain without any possibilities of reverting. In case your transaction is no longer valid it will stop being broadcasted to builders at no cost for the user.
```

Comment:
```suggestion
- **[No reverts](https://rpc.mevblocker.io/noreverts)** - focused on getting your transaction included on-chain without any possibilities of reverting. In case your transaction is no longer valid it will stop being broadcasted to builders at no cost for the user.
```